Source/UI/NewUI/PaymentForm/Factory/Theme/BillingForm/Theme+BillingInput.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Theme+BillingInput.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 10/08/2022.
+//
+
+import UIKit
+
+public extension Theme {
+    
+    /// Theme generated Billing Input Style
+    struct ThemeBillingInput: CellTextFieldStyle {
+        public var textfield: ElementTextFieldStyle
+        public var isMandatory: Bool
+        public var backgroundColor: UIColor = .clear
+        public var title: ElementStyle?
+        public var mandatory: ElementStyle?
+        public var hint: ElementStyle?
+        public var error: ElementErrorViewStyle?
+    }
+    
+    /// Create a Billing Input from provided themed components
+    func buildBillingInput(textField: ThemeTextField,
+                           isMandatory: Bool,
+                           title: ThemeTitle,
+                           mandatory: ThemeMandatory? = nil,
+                           subtitle: ThemeSubtitle? = nil,
+                           error: ThemeError? = nil) -> ThemeBillingInput {
+        ThemeBillingInput(textfield: textField,
+                          isMandatory: isMandatory,
+                          title: title,
+                          mandatory: mandatory,
+                          hint: subtitle,
+                          error: error)
+    }
+    
+    /// Create a Billing Input from provided content
+    func buildBillingInput(text: String,
+                           placeholder: String = "",
+                           isNumbericInput: Bool,
+                           isMandatory: Bool,
+                           title: String,
+                           subtitle: String? = nil,
+                           subtitleImage: UIImage? = nil,
+                           isRequiredText: String? = nil,
+                           errorText: String? = nil,
+                           errorImage: UIImage? = nil) -> ThemeBillingInput {
+        let subtitleText = subtitle ?? ""
```

Comment:
Question : if the default value for subtitleText is empty string should we just restrict the buildBillingInput to pass as this ? `subtitle: String = ""`

Reply:
We're actually using nil to decide if we should even show the component or not. THere's a task in the backlog to remove that empty space when component is not shown.

So outcome for different inputs would be:
- subtitle (or other optional Strings) => show component as expected
- nil => title is followed by TextField with standard padding
- "" (empty String) => element was purposely requested to be present, so will have double padding between title and textfield (title => 0 height subtitle => textfield)